package.json

```diff
@@ -149,7 +149,7 @@
     "less": "2.7.1",
     "less-loader": "2.2.3",
     "lodash": "3.10.1",
-    "marked": "0.3.6",
+    "markdown-it": "8.3.2",
```

Comment:
There are a few differences that I'm seeing between `marked`/`markdown-it`, using the following markdown:

```
Ordered

1. Lorem ipsum dolor sit amet
2. Consectetur adipiscing elit
3. Integer molestie lorem at massa


1. You can use sequential numbers...
1. ...or keep all the numbers as `1.`

Start numbering with offset:

57. foo
1. bar

Autoconverted link https://github.com/nodeca/pica (enable linkify to see)

```

I'm seeing the following with `marked`:
<img width="573" alt="screen shot 2017-08-22 at 3 02 04 pm" src="https://user-images.githubusercontent.com/627123/29582490-e98e0c70-874a-11e7-8733-88ce456ccda7.png">


And the following with `markdown-it`:
<img width="580" alt="screen shot 2017-08-22 at 3 01 44 pm" src="https://user-images.githubusercontent.com/627123/29582473-da342a52-874a-11e7-8d30-98be861fa934.png">

I believe that `markdown-it` is doing the numbering "correctly" or it at least matches what GitHub does; however, I figured this should at least be noted somewhere.

Regarding the auto-linking, this is something that `marked` used to automatically include, do we want to support this?